llvm/test/ExecutionEngine/JITLink/AArch32/ELF_static_thumb_reloc.s

```diff
@@ -95,6 +96,34 @@ data_symbol:
 
 	.text
 
+# CHECK-TYPE: {{[0-9a-f]+}} R_ARM_THM_MOVW_PREL_NC external_func
+# CHECK-INSTR: 	00000014 <movw_prel>:
+# CHECK-INSTR: 	      14: f240 0000     movw    r0, #0x0
+# jitlink-check: decode_operand(movw_prel, 1) = \
+# jitlink-check:              ((external_func - next_pc(movw_prel) + 4)&0x0000ffff)
```

Comment:
I'd still like to know where the `+ 4` is coming from. Shouldn't `next_pc()` provide the correct offset already? Isn't the relocation value relative to the next instruction? But I could do it post-review if necessary.

Reply:
Removing the `next_pc()` and `+ 4` solved the issue. Apparently I was using it to reference the address itself so I removed it now.

Reply:
Perfect, thanks!